metabase/metabase
# update permissions help

Epic https://github.com/metabase/metabase/issues/29491

### Description

Update permissions help sidebar content.

### How to verify

1. As an admin go to http://localhost:3000/admin/permissions
2. Check data permissions help
3. Check collection permissions help
4. Check application permissions help (EE-only)

### Demo

<img width="1362" alt="Screen Shot 2023-06-07 at 9 56 41 PM" src="https://github.com/metabase/metabase/assets/14301985/632aa34f-e237-4e5e-91cb-ce683f4dc110">

### Checklist
